bin/setup

```diff
@@ -117,7 +119,7 @@ verify_ruby_version (){
 verify_bundled_gems (){
   echo 'Updating gems...'
   {
-    (gem list -i bundler || gem install bundler) && bundle install --binstubs=./bundle/bin
+    (gem list -i -v ${BUNDLER_VERSION} bundler || gem install bundler -v ${BUNDLER_VERSION}) && bundle _${BUNDLER_VERSION}_ install --binstubs=./bundle/bin
```

Comment:
TIL `bundle _version_ install`

Reply:
yeah it is some weird syntax, but has saved me in some tough spots before